src/data/commonsense_dataset.py

```diff
@@ -33,7 +34,7 @@ def __init__(
         self.max_target_len = max_target_len or max_context_len
 
         self.bos_token = self.context_tokenizer.bos_token
-        self.eos_token = self.context_tokenizer.eos_token
+        self.eos_token = self.context_tokenizer.eos_token if mode == "blenderbot" else ""
```

Comment:
All tokenizers have `eos_token`, but only `blenderbot` contains `bos_token`. You can debug dataset implementations with `scripts.print_dataset` script.